2fa/v1/openapi.yaml

```diff
@@ -65,9 +76,13 @@ paths:
             $ref: '#/definitions/ErrorResponse'
     post:
       tags:
-        - 2fa-application-service
-      summary: Creates a 2fa application
-      description: Creates an application instance. Values that are not defined will be populated with default values.
+        - "Application Service"
+      summary: Create 2fa application
+      description: |
+        Creates a 2fa application instance. You can either:
```

Comment:
new line needed in order to have the items processed as expected.

Reply:
OK done!